theme/src/components/live-code.js

```diff
@@ -60,6 +60,8 @@ function LiveCode({code, language, noinline}) {
             style={{
               fontFamily: theme.fonts.mono,
               fontSize: '85%',
+              borderBottomLeftRadius: '6px',
+              borderBottomRightRadius: '6px',
```

Comment:
Should we use theme variables instead? 
```suggestion
              borderBottomLeftRadius: theme.radii[2],
              borderBottomRightRadius: theme.radii[2],
```

Reply:
You know, I glanced at the imports to see if the theme was available and skipped over the hook 🙈